onnxruntime/core/common/logging/sinks/ostream_sink.cc

```diff
@@ -65,7 +65,11 @@ void OStreamSink::SendImpl(const Timestamp& timestamp, const std::string& logger
 #ifdef _WIN32
 void WOStreamSink::SendImpl(const Timestamp& timestamp, const std::string& logger_id, const Capture& message) {
   // operator for formatting of timestamp in ISO8601 format including microseconds
-  using date::operator<<;
+#if __cplusplus >= 202002L
```

Comment:
you can use `timestamp_ns::operator<<` here to reduce the ifdefs.

the ifdefs here will need to be updated to work for Windows:
https://github.com/microsoft/onnxruntime/blob/c5087b9b58f929c37f5798c266a8b622b09bed82/include/onnxruntime/core/common/logging/logging.h#L61-L68